synapse/storage/databases/main/registration.py

```diff
@@ -53,6 +53,7 @@ class TokenLookupResult:
         valid_until_ms: The timestamp the token expires, if any.
         token_owner: The "owner" of the token. This is either the same as the
             user, or a server admin who is logged in as the user.
+        token_used: True if this token was used at least once in a request.
```

Comment:
it might be worth adding a comment here to note that `get_user_by_access_token` is cached, so this field might be out of date.

Reply:
Done